aten/src/ATen/native/TopKImpl.h

```diff
@@ -49,36 +49,60 @@ void topk_impl_loop(
       if (largest) {
```

Comment:
You'd better make sure `use_partial_sort` would be `false` if stable flag is on.

Reply:
> You'd better make sure use_partial_sort would be false if stable flag is on.

I think we would like to stable sort in the case of `auto use_partial_sort = k * 64 <= n` as well (i.e., stable sort regardless of `k` and input tensor dim `n`.